### ISSUE TITLE
Local project to be a package itself

### ISSUE THREAD

**Issue by guybedford** [jspm/jspm-cli#855]
And reflect config from package.json.

**Comment by robwormald** [comment]
making a note for myself here to add a prompt to the init setup to automate this, including adding it to the generated config.js file, and if using TS transpiler, adding the appropriate default extension.

**Comment by peteruithoven** [comment]
This would make it easier to refer to all the code in a project (but not dependencies in jspm_modules) in the packages configuration?

**Comment by guybedford** [comment]
Yes, the idea is to bring all the project configuration under a single package config.

**Comment by guybedford** [comment]
Plugin prompts something like - 

```
Would you like to install loader plugins? [no] y
Enter plugins to install [css, json]: 
```

**Comment by peteruithoven** [comment]
Like!

**Comment by guybedford** [comment]
Released in jspm 0.17 beta.
